### PR TITLE
added download instructions to yun [MKC-497]

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
We need to move the Yun downloads from the software page. This PR opens a new tutorial in the Yún space with the information.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/02.hero/boards/yun-rev2/tutorials/yun-download-os/content.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Follow the steps in the [Yún Sys Upgrade tutorial](./yun-sys-upgrade).
+
+### Advanced Installation Procedure
+
+This procedure is only recommended to advanced users who wish to completely re-flash the Yún including its U-Boot bootloader. [These instructions](./yun-u-boot-reflash) on reflashing the base images are for reference only. Following them will void your Yún's warranty.
+
+<DownloadLink href="/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip" title="Download" />
+
+[Packages list for Yún](//downloads.arduino.cc/openwrtyun/1/packages/index.html)
+[List of changes](https://github.com/arduino/openwrt-yun/blob/master/ChangeLog)
```
Comment (jhansson-ard) [inline content/hardware/02.hero/boards/yun-rev2/tutorials/yun-download-os/content.md:33]:
```suggestion

- [Packages list for Yún](//downloads.arduino.cc/openwrtyun/1/packages/index.html)
- [List of changes](https://github.com/arduino/openwrt-yun/blob/master/ChangeLog)

```

File: content/hardware/02.hero/boards/yun-rev2/tutorials/yun-download-os/content.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### Easy Installation Procedure (Recommended)
+
+Follow the steps in the [Yún Sys Upgrade tutorial](./yun-sys-upgrade).
+
+### Advanced Installation Procedure
+
+This procedure is only recommended to advanced users who wish to completely re-flash the Yún including its U-Boot bootloader. [These instructions](./yun-u-boot-reflash) on reflashing the base images are for reference only. Following them will void your Yún's warranty.
+
+<DownloadLink href="/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip" title="Download" />
```
Comment (jhansson-ard) [inline content/hardware/02.hero/boards/yun-rev2/tutorials/yun-download-os/content.md:30]: @sebromero not sure how we make this one visible. Any thoughts?

File: content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+### Easy Installation Procedure (Recommended)
+
+Follow the steps in the [Yún Sys Upgrade tutorial](./yun-sys-upgrade).
+
+### Advanced Installation Procedure
+
+This procedure is only recommended to advanced users who wish to completely re-flash the Yún including its U-Boot bootloader. [These instructions](./yun-u-boot-reflash) on reflashing the base images are for reference only. Following them will void your Yún's warranty.
+
+<DownloadLink href="/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip" title="Download" />
```
Comment (karlsoderby) [inline content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md:30]: I don't think this element renders at all. Maybe just turn this into a link?

File: content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+[Packages list for Yún](//downloads.arduino.cc/openwrtyun/1/packages/index.html)
```
Comment (karlsoderby) [inline content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md:32]:
```suggestion
- [Packages list for Yún](//downloads.arduino.cc/openwrtyun/1/packages/index.html)

```

File: content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+[List of changes](https://github.com/arduino/openwrt-yun/blob/master/ChangeLog)
```
Comment (karlsoderby) [inline content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md:33]:
```suggestion
- [List of changes](https://github.com/arduino/openwrt-yun/blob/master/ChangeLog)
```

File: content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md (hunk 3 above)
Comment (jhansson-ard) [reply content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md:30]: How do we do that? Just `<a href="/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip" title="Download" />` ?

Comment (karlsoderby) [reply content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md:30]: I think just a regular link, like `[Download](https://downloads.arduino.cc/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip)`, will be enough!

File: content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+### Easy Installation Procedure (Recommended)
+
+Follow the steps in the [Yún Sys Upgrade tutorial](./yun-sys-upgrade).
+
+### Advanced Installation Procedure
+
+This procedure is only recommended to advanced users who wish to completely re-flash the Yún including its U-Boot bootloader. [These instructions](./yun-u-boot-reflash) on reflashing the base images are for reference only. Following them will void your Yún's warranty.
+
+<DownloadLink href="/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip" title="Download" />
```
Comment (karlsoderby) [inline content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md:30]:
```suggestion
- [Download](https://downloads.arduino.cc/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip)
```

**Review by karlsoderby — APPROVED** [review]

**Review by sebromero — APPROVED** [review]
LGTM

**Pull request merged**